apps/financial-aid/web-osk/src/components/Login/login.tsx

```diff
@@ -11,53 +12,59 @@ interface Props {
 const Login = ({ headline, about }: Props) => {
   const router = useRouter()
 
-  const apiLoginRoute = '/api/auth/login'
+  const apiLoginRoute = router.query.id
+    ? `/api/auth/login?applicationId=${router.query.id}&nationalId=`
+    : '/api/auth/login?nationalId='
```

Comment:
Should we concat `nationald=` where use them below?

And should we have these routes as const in this file?

Reply:
Changed to two different ones for the time being, while we are still using fake users. What about that?